code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2666,96 +2666,979 @@ Follow by example and make good judgement based on length which list to include
+		name = "Tattoo (Jaeger, Right Eye)"
+		icon_state = "tat_jaeger_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_joosten_left
+		name = "Tattoo (Joosten, Left Eye)"
+		icon_state = "tat_joosten_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_joosten_right
+		name = "Tattoo (Joosten, Right Eye)"
+		icon_state = "tat_joosten_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_miller_left
+		name = "Tattoo (Miller, Left Eye)"
+		icon_state = "tat_miller_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_miller_right
+		name = "Tattoo (Miller, Right Eye)"
+		icon_state = "tat_miller_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_montana_left
+		name = "Tattoo (Montana, Left Face)"
+		icon_state = "tat_montana_l"
+		body_parts = list("head")
+		species_allowed = list("Human")
+
+	tat_montana_right
+		name = "Tattoo (Montana, Right Face)"
+		icon_state = "tat_montana_r"
+		body_parts = list("head")
+		species_allowed = list("Human")
+
+	tat_nightling
+		name = "Tattoo (Nightling, Back)"
+		icon_state = "tat_nightling"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_pawsocks
+		name = "Tattoo (Pawsocks, All)"
+		icon_state = "pawsocks"
+		body_parts = list("l_leg","r_leg","l_arm","r_arm","l_hand","r_hand","groin","chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+		chest
+			name = "Tattoo (Pawsocks, Torso)"
+			body_parts = list("chest")
+
+		groin
+			name = "Tattoo (Pawsocks, Groin)"
+			body_parts = list("groin")
+
+		left_arm
+			name = "Tattoo (Pawsocks, Left Arm)"
+			body_parts = list("l_arm")
+
+		right_arm
+			name = "Tattoo (Pawsocks, Right Arm)"
+			body_parts = list("r_arm")
+
+		left_hand
+			name = "Tattoo (Pawsocks, Left Hand)"
+			body_parts = list("l_hand")
+
+		right_hand
+			name = "Tattoo (Pawsocks, Right Hand)"
+			body_parts = list("r_hand")
+
+		left_leg
+			name = "Tattoo (Pawsocks, Left Leg)"
+			body_parts = list("l_leg")
+
+		right_leg
+			name = "Tattoo (Pawsocks, Right Leg)"
+			body_parts = list("r_leg")
+
+		left_foot
+			name = "Tattoo (Pawsocks, Left Foot)"
+			body_parts = list("l_foot")
+
+		right_foot
+			name = "Tattoo (Pawsocks, Right Foot)"
+			body_parts = list("r_foot")
+
+	tat_silverburgh_left
+		name = "Tattoo (Silverburgh, Left Leg)"
+		icon_state = "tat_silverburgh"
+		body_parts = list("l_leg")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_silverburgh_right
```

Comment:
Metal Gear Reference

Reply:
> This is one of those things where it was actually named such as-is and can even be found named so in quite a few other servers that have them.
I wish to understand what's the risk/consequence/problem in having a tattoo be named, on the menu, POSSIBLY after a character, if you would care to grant me such a thing?

> Regardless, I can try to rename it something else.